src/utils/components/HardwareDevices/HardwareDevicesHeadlessMode.tsx

```diff
@@ -16,9 +16,7 @@ const HardwareDevicesHeadlessMode: FC<HardwareDevicesHeadlessModeProps> = ({ onS
   const { t } = useKubevirtTranslation();
 
   const devices = vm?.spec?.template?.spec?.domain?.devices;
-  const [isChecked, setIsChecked] = useState<boolean>(
-    devices?.hasOwnProperty('autoattachGraphicsDevice') && !devices?.autoattachGraphicsDevice,
-  );
+  const [isChecked, setIsChecked] = useState<boolean>(devices?.autoattachGraphicsDevice === false);
```

Comment:
I think isHeadlessModeVMI(vm?.spec?.template) should work the same, can you check?

Reply:
u can use isHeadlessModeVMI ?

Reply:
The helper has been made more generic and moved to `selectors.ts`